Examples/SwiftExample/SwiftExample/AppDelegate.swift

```diff
@@ -7,20 +7,51 @@
 //
 
 import UIKit
+import UserNotifications
 import Purchases
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
 
+    func application(_ application: UIApplication,
+                     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
 
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        
         Purchases.debugLogsEnabled = true
         Purchases.configure(withAPIKey: <your_public_sdk_key_here>)
-        
+
+        Purchases.shared.setAttributes(["favorite_cat" : "garfield"])
+        requestPushNotificationsPermisssions()
```

Comment:
What do you think about adding a comment here to explain what this does? I can see a lot of people just copy pasting

Reply:
definitely a good idea, will do.

Reply:
@vegaro 3f05a6ef7865112e592e75642daf053507b5d393 thoughts?